panic: send on closed channel

```
panic: send on closed channel

goroutine 15413 [running]:
github.com/pion/ion-sfu/pkg.(*WebRTCSender).receiveRTCP(0xc002572e40)
	/go/src/github.com/pion/ion-sfu/pkg/sender.go:151 +0x18a
created by github.com/pion/ion-sfu/pkg.NewWebRTCSender
	/go/src/github.com/pion/ion-sfu/pkg/sender.go:65 +0x1c8
```

--- a/pkg/sender.go
+++ b/pkg/sender.go
@@ -134,7 +134,7 @@ func (s *WebRTCSender) Close() {
 func (s *WebRTCSender) receiveRTCP() {
 	for {
 		pkts, err := s.sender.ReadRTCP()
-		if err == io.ErrClosedPipe || err == io.EOF {
+		if err == io.ErrClosedPipe || err == io.EOF || s.ctx.Err() != nil {
 			return
 		}
 

